Library/Homebrew/download_strategy.rb

```diff
@@ -527,6 +529,26 @@ def curl(*args, **options)
   end
 end
 
+# Strategy for downloading a file from an GitHub Packages URL.
+#
+# @api public
+class CurlGitHubPackagesDownloadStrategy < CurlDownloadStrategy
+  attr_accessor :checksum, :name
+
+  private
+
+  def _fetch(url:, resolved_url:)
+    raise "Empty checksum" if checksum.blank?
+    raise "Empty name" if name.blank?
+
+    _, org, repo, = *url.match(GitHubPackages::URL_REGEX)
```

Comment:
```suggestion
    _, org, repo, = *url.match(GitHubPackages::URL_REGEX)
    repo.delete_prefix!("bottles-")
```
I prefer `bio`, and if there's consensus would accept `homebrew-bio`, but I'm not a fan of `bottles-bio`.

Reply:
@sjackman where is "bottles-bio" coming from here? I've not seen that reference anywhere.

Reply:
After removing `root_url` from the formula `brewsci/bio/ssm` and running
```bash
HOMEBREW_BOTTLE_DOMAIN=https://ghcr.io/v2/brewsci brew fetch ssm
```
it attempts to fetch from this URL which includes `bottles-bio`:
https://ghcr.io/v2/brewsci/bottles-bio/ssm/blobs/sha256:0cea947187c889476c284ed284120c4d33ea298140ced301a8da6b5aae78b352

Reply:
@sjackman Gotcha. I hadn't tested `HOMEBREW_BOTTLE_DOMAIN` here, I'll play around, thanks.

Reply:
@sjackman You need to use `HOMEBREW_BOTTLE_DOMAIN=https://ghcr.io/v2/brewsci/bio brew fetch ssm`

Reply:
Following a private Slack conversation, note that `HOMEBREW_BOTTLE_DOMAIN` and `root_url` have different semantics. The former specifies the organization, and the latter specifies both the organization and the repo.